content/post/online_hackathon_an_experiment.md

```diff
@@ -0,0 +1,33 @@
++++
+title = "Online Hackathon : An Experiment"
+slug = "first-online-hackathon-akshat"
+image = "img/akshat0047/h1.jpg"
+date = "2019-07-06T00:00:00"
+description = "Experience from my first online Hackathon"
+disableComments = false
+author = "Akshat Pande"
+tags = ['Hackathon','Coding','Bridgingo','First','Python']
+categories = ['Hackathon','Competition']
++++
+
+The first image which comes to a beginner's mind on hearing the word [Hackathon](https://en.wikipedia.org/wiki/Hackathon) is a group of people with professional ethical hacking skills who are racing towards a specific goal to win in a competition, even I thought this in a similar manner until my fellows from [ALiAS](https://asetalias.in) helped me to sort it out.
+
+In layman terms, Hackathon is just a coding competition where people in groups or individual work upon building something out of, either a list of projects/problem-statements or a project based on some theme given by the organizers. Also, participants are bound to a time limit usually ranging between 24-52 hours with proper arrangements of basic amenities for living. The same happens in an online mode too where you at your place race to the finish line.
```

Comment:
Adding a line about "availability of mentors there to help you out with your problems" seems like cherry on the top.

Reply:
Implemented